controllers/operandrequest/reconcile_operand.go

```diff
@@ -130,15 +130,33 @@ func (r *Reconciler) reconcileOperand(ctx context.Context, requestInstance *oper
 				continue
 			}
 
-			if csv.Status.Phase == olmv1alpha1.CSVPhaseFailed {
-				merr.Add(fmt.Errorf("the ClusterServiceVersion of Subscription %s/%s is Failed", namespace, operatorName))
-				requestInstance.SetMemberStatus(operand.Name, operatorv1alpha1.OperatorFailed, "", &r.Mutex)
+			if sub.Status.Install == nil || sub.Status.InstallPlanRef.Name == "" {
+				klog.Warningf("The Installplan for Subscription %s is not ready. Will check it again", sub.Name)
```

Comment:
Should we consider set the member status to `Installing` here since we skip it?

Reply:
Done